bin/generatePhosimInput.py

```diff
@@ -0,0 +1,85 @@
+"""
+Script to run the generation of phoSim instance catalogs
+"""
+from __future__ import with_statement, absolute_import, division, print_function
+import os
+from lsst.sims.catUtils.utils import ObservationMetaDataGenerator
+from desc.twinkles import TwinklesSky
+import argparse
+import pandas as pd
+from sqlalchemy import create_engine
+import time
+
+# Choose OpSim Output database
+OpSimDB = 'minion_1016_sqlite.db'
```

Comment:
@TomGlanzman Can you have a look at and comment on this proposed interface for running it in the pre-burner stage discussed at #313 ?